Improve TCO

Marchabar opened the pull request:
# Description

I have moved the TCO to Docusaurus and also updated it with the feedback we got on last tuesday's class. Added SLA costs, and updates on previous information.

# Checklist:
- [x] I have performed a self-review of my code running `npm run build` and `npm run serve` after pushing the code.
- [x] I have checked that there is no fragile information published unintentionally.

# Screenshot:
**New Index**

![image](https://github.com/ISPP-07/htld-doc/assets/100620336/45158549-6cbf-4b72-a36d-64bd637d023e)

alvgonfri reviewed (changes requested):
- I haven't seen anything related to this point of the issue description: "Include a TCO evolution (e.g.: 200€/month per 1000 users, if the number of users doubles, what happens to the operation costs)."
- Section 3.2 is repeated:
![image](https://github.com/ISPP-07/htld-doc/assets/73192598/4a5934fe-6df4-4bb9-b70e-d89caf609c68)
- The final cost information is missing at the GDPR section (I think you missed to include that info, which was written in the original document).

Marchabar commented:
> * I haven't seen anything related to this point of the issue description: "Include a TCO evolution (e.g.: 200€/month per 1000 users, if the number of users doubles, what happens to the operation costs)."
> * Section 3.2 is repeated:
>   ![image](https://private-user-images.githubusercontent.com/73192598/309424229-4a5934fe-6df4-4bb9-b70e-d89caf609c68.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkzNzc5MzUsIm5iZiI6MTcwOTM3NzYzNSwicGF0aCI6Ii83MzE5MjU5OC8zMDk0MjQyMjktNGE1OTM0ZmUtNmRmNC00YmI5LWI3MGUtZDg5Y2FmNjA5YzY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzAyVDExMDcxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEyMDFlMGE4ZGQzZjNlOGEwNjZjYTVhNTQyMzYyZDQxN2ZmZTE4NzY5MGZmNmI0M2Y1YzRmNTMzNzY5MDJmZDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9egxa3LjtI49QdV29O_9xEpjHceV-bvvvhQ-uBFGavY)
> * The final cost information is missing at the GDPR section (I think you missed to include that info, which was written in the original document).

I have changed what you requested. About the first point I already added it to the doc, but I think it was not in the correct place. Now I have moved it to section "3. Costes de operación mensual (Opex)". This is what I had before:

![image](https://github.com/ISPP-07/htld-doc/assets/100620336/03fc144d-d3e2-4078-81cb-26888eefb5cc)

alvgonfri reviewed (approved):
The requested changes have been addressed. The TCO evolution is clearer now.
